react-components/src/components/Image360CollectionContainer/Image360CollectionContainer.tsx

```diff
@@ -5,18 +5,18 @@ import { type ReactElement, useEffect, useRef } from 'react';
 import { useReveal } from '../RevealCanvas/ViewerContext';
 import { type Image360Collection } from '@cognite/reveal';
 import { useRevealKeepAlive } from '../RevealKeepAlive/RevealKeepAliveContext';
-import { type AddImageCollection360Options } from '../..';
+import { type AddImage360CollectionOptions } from '../Reveal3DResources/types';
 import {
   type ImageCollectionModelStyling,
   useApply360AnnotationStyling
 } from './useApply360AnnotationStyling';
 import { type Matrix4 } from 'three';
 
 type Image360CollectionContainerProps = {
-  addImageCollection360Options: AddImageCollection360Options;
+  addImageCollection360Options: AddImage360CollectionOptions;
```

Comment:
If it doesnt break anything, maybe it is also good to update the parameter's name to follow the same pattern.

Reply:
:+1: